fusor-ember-cli/app/routes/register-nodes.js

```diff
@@ -85,19 +85,19 @@ export default Ember.Route.extend({
     }).then((result) => {
       controller.set('ports', result.ports);
     }, (error) => {
-      return this.send('error', error, `Unable to load node ports. GET "${url}" failed with status code ${error.jqXHR.status}.`);
+      this.send('error', error, `Unable to load node ports. GET "${url}" failed with status code ${error.jqXHR.status}.`);
     });
   },
 
   loadIntrospectionTasks() {
     let controller = this.get('controller');
     let deploymentId = this.get('controller.deploymentId');
-    this.store.findRecord('deployment', deploymentId, {reload: true}).then(
+    return this.store.findRecord('deployment', deploymentId, {reload: true}).then(
```

Comment:
I was deliberately doing that, but I'll use catch from now on.  I think it's nice and clear.